aws_marketplace/using_algorithms/automl/AutoML_-_Train_multiple_models_in_parallel.ipynb

```diff
@@ -0,0 +1,635 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#Let us install necessary H2O library which you would use to load and inspect the model summary.\n",
+    "import sys\n",
+    "!{sys.executable} -m pip install http://h2o-release.s3.amazonaws.com/h2o/rel-wright/10/Python/h2o-3.20.0.10-py2.py3-none-any.whl"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#Import necessary libraries.\n",
+    "import boto3\n",
+    "import re\n",
+    "import os\n",
+    "import errno\n",
+    "import base64\n",
+    "import time\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "import urllib\n",
+    "from sagemaker import get_execution_role\n",
+    "import json\n",
+    "import uuid\n",
+    "import sagemaker\n",
+    "from time import gmtime, strftime\n",
+    "import urllib.request\n",
+    "from sagemaker import AlgorithmEstimator\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Step 1: Subscribe to AutoML algorithm from AWS Marketplace\n",
+    "\n",
+    "1. Open [H2O-3 Automl Algorithm listing from AWS Marketplace](https://aws.amazon.com/marketplace/pp/prodview-vbm2cls5zcnky?qid=1557245796960&sr=0-1&ref_=srh_res_product_title)\n",
+    "2. Read the **Highlights** section and then **product overview** section of the listing.\n",
+    "3. View **usage information** and then **additional resources**.\n",
+    "4. Note the supported instance types and specify the same in the following cell.\n",
+    "5. Next, click on **Continue to subscribe**.\n",
+    "6. Review **End user license agreement**, **support terms**, as well as **pricing information**.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "compatible_training_instance_type='ml.c5.4xlarge' \n",
+    "\n",
+    "compatible_inference_instance_type='ml.c5.2xlarge' "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "7. Next, \"Accept Offer\" button needs to be clicked only if your organization agrees with EULA, pricing information as well as support terms. Once **Accept offer** button has been clicked, specify compatible training and inference types you wish to use. \n",
+    "\n",
+    "**Notes**: \n",
+    "1. If **Continue to configuration** button is active, it means your account already has a subscription to this listing.\n",
+    "2. Once you click on **Continue to configuration** button and then choose region, you will see that a product ARN will appear. This is the algorithm ARN that you need to specify in your training job. However, for this notebook, the algorithm ARN has been specified in src/scikit_product_arns.py file and you do not need to specify the same explicitly."
```

Comment:
why are you calling the file with product arns as "scikit_product_arns" ? The filename is confusing as this notebook has nothing to do with scikit and seems like a copy-paste mistake?

Reply:
renamed the file